Rakefile

```diff
@@ -1,15 +1,18 @@
 require "rake"
 require "rake/testtask"
+require "rubocop/rake_task"
 require "bundler"
 
 Bundler::GemHelper.install_tasks
 
+RuboCop::RakeTask.new(:lint)
```

Comment:
There probably isn't any need to deviate from the default of rubocop here. Often our `lint` tasks do all the types of linting of an app and rubocop can be specific for just ruby linting.